CMSgov/cms-carts-seds
# [CMDCT-3976] ETL to add <11 mask to all forms

### Description

This ETL is a follow up to the [<11](https://github.com/Enterprise-CMCS/macpro-mdct-carts/pull/139751) PR work and expands upon the removal of certain calculations in the print view as [defined in this ticket](https://jiraent.cms.gov/browse/CMDCT-4014). It goes through and adds the <11 mask to the 2023 and 2022 prior forms for all states for the exact same questions and removes a few calculations in sections 3 and 5. You can see a live before and after of running this script in the deployed environment here:

https://github.com/user-attachments/assets/bd7b3594-8c67-4d7d-8ad7-24bdba058512



### Related ticket(s)


CMDCT-3976

---
### How to test


- `./run local` (To ensure a brand new, seeded database)
- In a new terminal run `DYNAMO_ENDPOINT=http://localhost:8000 dynamodb-admin` after the local environment has fully spun up.
- Navigate to localhost:8001 to see the local instance of dynamo.
- Confirm that for the report AL-2022, Section 3c questions 2-4 that in the print view entering 5 still shows as 5.
- In a new terminal, run `DYNAMODB_URL="http://localhost:8000" dynamoPrefix="local" node services/database/scripts/add-less-than-eleven-mask.js`
- Reload the 2022 report
- Navigate to Section 3c
- See that question 2-4 in Part 5 still show your original answer of 5
- Click the print button and then click print section at the bottom of the page
- See that the questions answer now shows <11
- The computed calculations will be hidden and no longer shown
- Navigate to localhost:8001 to see the local instance of dynamo.
- Confirm that for the report AL-2022, Section 3 that there there has been a mask added to that field.


Extra Credit
- Logout and sign in as an admin (cms.admin@test.com)
- Open the 2022 report and navigate to Section 3c
- See that all the fields also show <11 and computed values associated are hidden

---
### Pre-review checklist


---
### Pre-merge checklist


#### Review
